drv/stm32h7-eth/Cargo.toml

```diff
@@ -12,9 +12,10 @@ ipv4 = []
 ipv6 = []
 
 [dependencies]
+cfg-if = "0.1.10"
 cortex-m = "0.7"
 userlib = {path = "../../sys/userlib"}
-stm32h7 = {version = "0.13", default-features = false}
+stm32h7 = {version = "0.13"}
```

Comment:
You want to keep `default-features = false` here, the stm32 PAC crates turn on the `rt` feature by default, which is inappropriate unprivileged task code like this.

Reply:
OK, understood. In my $HOME/.cargo/registry/src/github.com-1ecc6299db9ec823/stm32h7-0.13.0/Cargo.toml file, I see:

```
[features]
default = []
```

So I thought that removing it like net/task/Cargo.toml was more right.

Reply:
They changed the default in 0.14, and I'm in the process of upgrading things. Thanks for future-proofing it. :-)